deployment/ansible/roles/open-apparel-registry.environment/tasks/main.yml

```diff
@@ -5,6 +5,12 @@
     regexp: "^AWS_PROFILE="
     line: "AWS_PROFILE={{ aws_profile }}"
 
+- name: Set OAR_SETTINGS_BUCKET
```

Comment:
Without this, you can't run `bootstrap` from within the virtual machine. It works within the context of `vagrant up`, but that's because we set it inside of the `Vagrantfile`.